examples/00-fluent/radiation_headlamp.py

```diff
@@ -0,0 +1,519 @@
+
+###############################################################################
+# Improve surface mesh
+# ~~~~~~~~~~~~~~~~~~~~
+# Improve the surface mesh.
+
+surface_mesh_gen.InsertNextTask(CommandName="ImproveSurfaceMesh")
+
+meshing.workflow.TaskObject["Improve Surface Mesh"].Execute()
+
+###############################################################################
+# Describe geometry
+# ~~~~~~~~~~~~~~~~~
+# Describe geometry and define the fluid region.
+
+describe_geo = meshing.workflow.TaskObject["Describe Geometry"]
+describe_geo.Arguments.set_state(
+    {
+        "SetupType": "The geometry consists of both fluid and solid regions and/or voids",
+        "CappingRequired": "No",
+        "WallToInternal": "No",
+        "InvokeShareTopology": "No",
+        "Multizone": "No",
+    }
+)
+
+describe_geo.Execute()
+
+###############################################################################
+# Update boundaries
+# ~~~~~~~~~~~~~~~~~
+# Update the boundaries.
+
+update_bc = meshing.workflow.TaskObject["Update Boundaries"]
+update_bc.Arguments.set_state(
+    {
+        "BoundaryLabelList": ["rad-input"],
+        "BoundaryLabelTypeList": ["wall"],
+    }
+)
+
+update_bc.Execute()
+
+###############################################################################
+# Create fluid region
+# ~~~~~~~~~~~~~~~~~~~
+# Create the fluid region.
+
+create_regions = meshing.workflow.TaskObject["Create Regions"]
+create_regions.Arguments.set_state({"NumberOfFlowVolumes": 1})
+
+create_regions.Execute()
+
+###############################################################################
+# Update regions
+# ~~~~~~~~~~~~~~
+# Update the regions.
+
+meshing.workflow.TaskObject["Update Regions"].Execute()
+
+###############################################################################
+# Boundary layers
+# ~~~~~~~~~~~~~~~~~~~
+# Do not add boundary layers and proceed to the next task.
+
+add_boundary_layers = meshing.workflow.TaskObject["Add Boundary Layers"]
+add_boundary_layers.Arguments.set_state({"AddChild": "no"})
+
+add_boundary_layers.Execute()
+
+###############################################################################
+# Generate volume mesh
+# ~~~~~~~~~~~~~~~~~~~~
+# Generate the volume mesh, which consists of setting properties for the
+# volume mesh.
+
+volume_mesh_gen = meshing.workflow.TaskObject["Generate the Volume Mesh"]
+volume_mesh_gen.Arguments.set_state(
+    {
+        "VolumeMeshPreferences": {
+            "PolyFeatureAngle": 40,
+        },
+    },
+)
+volume_mesh_gen.Execute()
+
+###############################################################################
+# Check mesh in meshing mode
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Check the mesh in meshing mode.
+
+meshing.tui.mesh.check_mesh()
+
+###############################################################################
+# Save mesh file
+# ~~~~~~~~~~~~~~
+# Save the mesh file (``headlamp.msh.h5``).
+
+meshing.tui.file.write_mesh("headlamp.msh.h5")
```

Comment:
Starting with 2024 R1, this TUI call will be able to be replaced by `meshing.File.WriteMesh(FileName=)`, though more wordy and not using snake case

Reply:
Would you recommend changing this line? I developed this on 2023 R2.

Reply:
No, that was more of a note for myself and others. Currently our doc builds use Fluent 2023 R2 as well, so you used the right command there. I believe there is no non-TUI equivalent in this version.